pose_thumbnails/core.py

```diff
@@ -546,6 +549,11 @@ def _execute(self, context, auto_key: bool):
         return {'FINISHED'}
 
     def modal(self, context, event):
+        if self._target_state == 'ABORTED':
```

Comment:
When is `'ABORTED'` used? Please include all relevant code changes in the pull request.

Reply:
Originally this pulled in to match your 2.8 branch, but then I saw that relied upon using handlers that don't exist before 2.8 (undo_pre and redo_pre) - looks like I forgot to back out these parts of that merge.

But a more important question, is there an equivalent way this should be done in 2.7X, to avoid undo/redo from trying to access invalid pointers?

Other changes largely updated locally, will push once I have more direction on this part.